### PR TITLE
[WIP] storage key

### PR DESCRIPTION
**Pull request by tabaktoni**
## Motivation and Resolution

- enforce format on rpc storage key  
- simple getStorageAt tests
- remove estFee version expectation

## Checklist:

- [x] Performed a self-review of the code
- [x] Rebased to the last commit of the target branch (or merged it into my branch)
- [ ] Linked the issues which this PR resolves
- [ ] Documented the changes in code (API docs will be generated automatically)
- [x] Updated the tests
- [x] All tests are passing

### CODE REVIEW

File: __mocks__/cairo/helloSierra/hello.cairo [hunk 1]
```diff
@@ -328,6 +329,22 @@ mod HelloStarknet {
         testbet::read()
     }
 
+    //#[external]
```
Comment (ivpavici) [inline __mocks__/cairo/helloSierra/hello.cairo:332]: leftover or?

Comment (tabaktoni) [reply __mocks__/cairo/helloSierra/hello.cairo:332]: Yea, that method was required by ex compiler to include the struct in ABI now it is fixed.

File: __tests__/cairo1.test.ts [hunk 2]
```diff
@@ -149,6 +150,47 @@ describeIfDevnet('Cairo 1 Devnet', () => {
       expect(status).toBe(123n);
     });
 
+    test('Cairo1 simple getStorageAt variables retreaval', async () => {
```
Comment (ivpavici) [inline __tests__/cairo1.test.ts:153]:
```suggestion
    test('Cairo1 simple getStorageAt variables retrieval', async () => {
```

File: __tests__/account.test.ts [hunk 3]
```diff
@@ -632,8 +647,8 @@ describe('deploy and test Wallet', () => {
         expect(value).toMatchSchemaRef('EstimateFee');
       });
       expect(estimatedFeeBulk.length).toEqual(2);
-      expect(innerInvokeEstFeeSpy.mock.calls[0][1].version).toBe(feeTransactionVersion);
-      innerInvokeEstFeeSpy.mockClear();
+      // expect(innerInvokeEstFeeSpy.mock.calls[0][1].version).toBe(feeTransactionVersion);
```
Comment (ivpavici) [inline __tests__/account.test.ts:650]: maybe good to add some TODO here?

File: __tests__/account.test.ts [hunk 4]
```diff
@@ -733,16 +748,16 @@ describe('deploy and test Wallet', () => {
...
       const result = await account.estimateInvokeFee({
         contractAddress: ddc1.deploy.address,
         entrypoint: 'increase_balance',
         calldata: [100],
       });
 
       expect(result).toMatchSchemaRef('EstimateFee');
-      expect(innerInvokeEstFeeSpy.mock.calls[0][1].version).toBe(feeTransactionVersion);
-      innerInvokeEstFeeSpy.mockClear();
+      // expect(innerInvokeEstFeeSpy.mock.calls[0][1].version).toBe(feeTransactionVersion);
```
Comment (ivpavici) [inline __tests__/account.test.ts:759]: maybe good to add some TODO here?

**Review by penovicp — APPROVED** [review]

File: __tests__/account.test.ts (hunk 4 above)
Comment (tabaktoni) [reply __tests__/account.test.ts:761]: ed2c5ee5f6acf2f2ae8429878a5c59962027a93e

**Pull request merged**